backend/core/views.py

```
@@ -4813,3 +4813,65 @@ class FindingViewSet(BaseModelViewSet):
+                incident=instance,
+                entry=f"{previous_instance.get_severity_display()}->{instance.get_severity_display()}",
+                entry_type=TimelineEntry.EntryType.SEVERITY_CHANGED,
+                author=self.request.user,
+                timestamp=now(),
+            )
+
+        return super().perform_update(serializer)
+
+
+class TimelineEntryViewSet(BaseModelViewSet):
+    model = TimelineEntry
```

Ruff (0.8.2)

`TimelineEntry` may be undefined, or defined from star imports

(F405)